code/ui/tttpanel/TTTPanel.cs

```diff
@@ -23,6 +23,16 @@ public TTTPanel(Panel parent = null) : base(parent)
             IsShowing = false;
         }
 
+        public void Open()
+        {
+            IsShowing = true;
+        }
+
+        public void Close()
+        {
+            IsShowing = false;
+        }
+
```

Comment:
Might want to add an issue for a refactor, we can use these methods, then hide away `IsShowing` within the `TTTPanel` class

Reply:
This should be addressed in the UI Rework: https://github.com/TTTReborn/tttreborn/issues/195